src/main/java/hudson/plugins/ec2/EC2AbstractSlave.java

```diff
@@ -98,6 +98,7 @@
     public List<EC2Tag> tags;
     public final String cloudName;
     public AMITypeData amiType;
+    private String instanceType = null;
```

Comment:
Per consistency with the above, and readability, initializing variables in Java is unneeded FWIW. What's written above is what happens anyway.